src/db/index.ts

```diff
@@ -0,0 +1,39 @@
+import { DataTypes } from 'sequelize';
+import UsersAttributes from '../types/usersAttributes';
+import sequelize from './connection';
+
+const Users = sequelize.define<UsersAttributes>('users', {
```

Comment:
model name is preferred to be singular, please check [Sequelize singular-vs-plural](https://sequelize.org/docs/v6/other-topics/naming-strategies/#singular-vs-plural)
```suggestion
const User = sequelize.define<UsersAttributes>('user', {
```

Reply:
done